Code/src/main/java/nl/utwente/viskell/haskell/env/CatalogFunction.java

```diff
@@ -33,6 +37,31 @@
         this.isCommon = isCommon;
     }
 
+    @Override
+    public Map<String, Object> toBundleFragment() {
+        Map<String, Object> bundleFragment = new HashMap<>();
+        bundleFragment.put(Bundleable.KIND, this.getClass().getSimpleName());
+        bundleFragment.put("name", name);
+        bundleFragment.put("signature", signature.toString());
+        bundleFragment.put("category", category);
+        bundleFragment.put("documentation", documentation);
+        bundleFragment.put("isConstructor", isConstructor);
+        bundleFragment.put("isCommon", isCommon);
```

Comment:
Seems like the HaskellCatalog is lacking a method for that, so the ugly way to do this now is:
` toplevelPane.getGhciSession().getCatalog().getByPredicate(fn -> fn.getName().equals(name)).iterator().next();`